diff --git a/node_modules/stylelint-z-index-value-constraint/index.js b/node_modules/stylelint-z-index-value-constraint/index.js
index 1617c19..8ebe669 100644
--- a/node_modules/stylelint-z-index-value-constraint/index.js
+++ b/node_modules/stylelint-z-index-value-constraint/index.js
@@ -68,6 +68,7 @@ module.exports = stylelint.createPlugin(
 
         if (options.max && Math.abs(value) > options.max) {
           stylelint.utils.report({
+				messageArg: [options.max],
             ruleName,
             result,
             node: decl,
@@ -80,6 +81,7 @@ module.exports = stylelint.createPlugin(
         if (options.min && Math.abs(value) < options.min) {
           stylelint.utils.report({
             ruleName,
+				messageArg: [options.min],
             result,
             node: decl,
             message: messages.smallerThanMin(
