diff --git a/node_modules/stylelint-declaration-block-no-ignored-properties/LICENSE b/node_modules/stylelint-declaration-block-no-ignored-properties/LICENSE
index f090ed7..f3418c5 100644
--- a/node_modules/stylelint-declaration-block-no-ignored-properties/LICENSE
+++ b/node_modules/stylelint-declaration-block-no-ignored-properties/LICENSE
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2018 Krister Kari
+Copyright (c) 2023 Marc-André Charpentier
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
diff --git a/node_modules/stylelint-declaration-block-no-ignored-properties/index.js b/node_modules/stylelint-declaration-block-no-ignored-properties/index.js
index 89b9faa..e9857ca 100644
--- a/node_modules/stylelint-declaration-block-no-ignored-properties/index.js
+++ b/node_modules/stylelint-declaration-block-no-ignored-properties/index.js
@@ -215,6 +215,7 @@ const rule = (actual) => {
             }
 
             report({
+					messageArgs: [node.prop, decl.toString()],
               message: messages.rejected(node.prop, decl.toString()),
               node,
               result,
